src/main/java/frc/robot/subsystems/gpm/Shooter.java

```diff
@@ -43,239 +44,244 @@ public class Shooter extends SubsystemBase {
 	private static final double MOI_SHAFT = MOI_COLSON * 4;
 
 	// top motor
-    private final CANSparkFlex topMotor = new CANSparkFlex(ShooterConstants.TOP_MOTOR_ID, MotorType.kBrushless);
-    private final RelativeEncoder topMotorEncoder = topMotor.getEncoder();
+	private final CANSparkFlex leftMotor = new CANSparkFlex(ShooterConstants.LEFT_MOTOR_ID, MotorType.kBrushless);
+	private final RelativeEncoder leftMotorEncoder = leftMotor.getEncoder();
 	/** PID controller uses RPM as input and outputs motor power */
-    private final PIDController topPID = new PIDController(P, I, D);
-	private FlywheelSim topFlywheelSim;
-	private double topMotorSpeedSim;
-	private double topPower = 0.0;
-
-	// bottom motor
-    private final CANSparkFlex bottomMotor = new CANSparkFlex(ShooterConstants.BOTTOM_MOTOR_ID, MotorType.kBrushless);
-    private final RelativeEncoder bottomMotorEncoder = bottomMotor.getEncoder();
+	private final PIDController leftPID = new PIDController(P, I, D);
+	private FlywheelSim leftFlywheelSim;
+	private double leftMotorSpeedSim;
+	private double leftPower = 0.0;
+
+	// bottrightor
+	private final CANSparkFlex rightMotor = new CANSparkFlex(ShooterConstants.RIGHT_MOTOR_ID, MotorType.kBrushless);
+	private final RelativeEncoder rightMotorEncoder = rightMotor.getEncoder();
 	/** PID controller uses RPM as input and outputs motor power */
-    private final PIDController bottomPID = new PIDController(P, I, D);
-	private FlywheelSim bottomFlywheelSim;
-	private double bottomMotorSpeedSim;
-	private double bottomPower = 0.0;
-	
-    // TODO: TUNE THIS
-    private final SimpleMotorFeedforward feedforward = new SimpleMotorFeedforward(S, V);
-	
-    public Shooter() {
+	private final PIDController rightPID = new PIDController(P, I, D);
+	private FlywheelSim rightFlywheelSim;
+	private double rightMotorSpeedSim;
+	private double rightPower = 0.0;
+
+	// TODO: TUNE THIS
+	private final SimpleMotorFeedforward feedforward = new SimpleMotorFeedforward(S, V);
+
+	public Shooter() {
 		// set the RPM tolerance of the PID controllers
-        topPID.setTolerance(TOLERANCE);
-        bottomPID.setTolerance(TOLERANCE);
+		leftPID.setTolerance(TOLERANCE);
+		rightPID.setTolerance(TOLERANCE);
 		// invert the bottom motor so +power sends the note out
```

Comment:
Change this comment and any other references to top/bottom motors.

Reply:
yer